bin/run-browser-test-env

```diff
@@ -47,6 +54,9 @@ Options:
 
 --azure Use the Azure emulator (Azurite) for simulating cloud storage for file
         upload and download. Also use localstack, since we need that for SES.
+
+--local Set BASE_URL to http://localhost:9999 to enable running
+        bin/run-browser-tests-local
```

Comment:
Suggestion: Since we have `bin/run-browser-tests` and bin/run-browser-tests-local`, can we have a `bin/run-browser-test-env-local` as well? Then we can remove the `--local` arg, set `BASE_URL` in `bin/run-browser-test-env-local`, and call `bin/run-browser-test-env` from that new file. What do you think?

Reply:
We could, but I think we'd end up duplicating a lot of the code in here?

Reply:
Sorry, didn't fully grok what you said at first.  Yeah, that's a good point, that might be cleaner.

Reply:
I think to split it out into a separate file, we will either need to pass a flag into run-browser-test-env anyway to tell it to run in local mode or add something to truth.sh to detect when a variable is set and not exit, as we'll be wanting to set a variable in run-browser-test-env-local. What do you think?

Reply:
I see - if we would need the `local_mode` flag regardless, then I'll leave it up to you. I guess it's not so different to run `bin/run-browser-test-env --local` vs `bin/run-browser-test-env-local`

Reply:
Alright.  I think for now I'll just leave it.  Suppose we could always have `bin/run-browser-test-env-local` that calls `bin/run-browser-test-env --local` :)